scripts/hooks/render-item-5e-sheet.mjs

```diff
@@ -4,7 +4,10 @@ import { FormulaGroupConfig } from "../apps/formula-group-config.mjs";
 Hooks.on("renderItemSheet5e", (itemSheet, html, _) => {
     // Add formula group config button
     const tooltip = game.i18n.localize(`${MODULE_NAME}.FORMULA-GROUP.DialogTitle`);
-    html.find(".tab.details .damage-header").prepend(`<a title="${tooltip}" class="config-formula-groups"><i class="fas fa-tasks"></i></a>`);
+    const damageHeader = html.find(".tab.details .damage-header");
+    damageHeader.wrap(`<div class="mre-damage-header-container">`);
```

Comment:
Possible this won't exist, e.g. for Tool items. Should this be option chained or is jquery smart enough to handle that?

Reply:
I think jQuery will handle that, but I will double check tomorrow.

Reply:
Yes, jQuery will just do nothing if those elements are not present.